### PR TITLE
[BB-9078] Styling fix

### PR DESCRIPTION
**Pull request by pkulkark**
This PR fixes the styling for long titles that should be truncated.

### CODE REVIEW

File: src/course-outline/card-header/CardHeader.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
   .item-card-header__title-btn {
     justify-content: flex-start;
     padding: 0;
-    width: fit-content;
+    width: 100%;
```
Comment (kaustavb12) [inline src/course-outline/card-header/CardHeader.scss]: @pkulkark 

Although this solves the issue at hand, it does make the edit button as well as the `Draft (Unpub..)` tag look quite odd and squished

Before change:
image

After change:
image

I would suggest the following changes which seems to solve our problem without affecting other elements:
```suggestion
    width: fit-content;
    max-width: 62%;
```

What do you think ? cc. @0x29a

Comment (pkulkark) [reply src/course-outline/card-header/CardHeader.scss]: @kaustavb12 I'm not sure that's going to work. With `width: fit-content` the truncated line gets wrapped which was the original issue. Maybe we can play around with max-width while keeping `width: 100%` but it feels like that's still going to throw off the styling one way or the other.  The oc instance studio seems to be down at the moment. Will test it once it's up and running and update.

File: src/course-outline/card-header/CardHeader.scss [hunk 2]
```diff
@@ -15,6 +15,7 @@
   .item-card-edit-icon {
     opacity: 0;
     transition: opacity .3s linear;
+    margin-right: 5px;
```
Comment (kaustavb12) [inline src/course-outline/card-header/CardHeader.scss]: For symmetry on both sides of the status badge
```suggestion
    margin-right: 0.5rem;
```

**Review by kaustavb12 — APPROVED** [review]
:+1:

- [x] I tested this: Tested in local
- [x] I read through the code

**Pull request merged**